CyclopsNuclearReactor/CyNukReactorSMLHelper.cs

```diff
@@ -0,0 +1,122 @@
+﻿namespace CyclopsNuclearReactor
+{
+    using SMLHelper.V2.Assets;
+    using SMLHelper.V2.Crafting;
+    using SMLHelper.V2.Handlers;
+    using UnityEngine;
+
+    internal class CyNukReactorSMLHelper : Buildable
+    {
+        private static readonly CyNukReactorSMLHelper main = new CyNukReactorSMLHelper();
+
+        private const string EquipmentLabelKey = "CyNukeRodsLabel";
+        public static string EquipmentLabel()
+        {
+            return Language.main.Get(EquipmentLabelKey);
+        }
+
+        private const string DepletedMessageKey = "CyNukeRodDepleted";
+        public static string DepletedMessage()
+        {
+            return Language.main.Get(DepletedMessageKey);
+        }
+
+        private const string OnHoverKey = "CyNukeOnHover";
+        public static string OnHoverText(int currentPower)
+        {
+            return Language.main.GetFormat(OnHoverKey, currentPower);
+        }
+
+        private const string OverLimitKey = "CyNukeOverLimit";
+        public static string OverLimit()
+        {
+            return Language.main.Get(OverLimitKey);
+        }
+
+        public static TechType TechTypeID { get; private set; }
+
+        public static void PatchSMLHelper()
+        {
+            main.Patch();
+        }
+
+        public CyNukReactorSMLHelper() : base("CyNukReactor", "Cyclops Nuclear Reactor", "A nuclear reactor re-designed to fit and function inside the Cyclops.")
+        {
+            OnFinishedPatching += AdditionalPatching;
+        }
+
+        public override TechGroup GroupForPDA { get; } = TechGroup.Cyclops;
+        public override TechCategory CategoryForPDA { get; } = TechCategory.Cyclops;
+
+        // TODO This needs an icon sprite
+        public override string AssetsFolder { get; } = "CyclopsNuclearReactor/Assets";
```

Comment:
This will require a `CyNukReactor.png` image inside the folder `.QMods/CyclopsNuclearReactor/Assets/` to serve as the icon for this buildable in the PDA.